062-prometheus-metrics-reporter.md

```diff
@@ -56,40 +56,66 @@ metricsConfig:
 ```
 
 Reporters will expose the following configurations:
```

Comment:
I think we need a property to control the bind address.  Default to binding all interfaces interface, but let the user specify a property if they need to be more restrictive.

Reply:
That's a good idea. Then we should probably have a `listener` config that defaults to `http://:8080` and remove the `port` config. WDYT?

Reply:
I made the change

Reply:
So the protocol part is ignored, right?  I worry that a user might mistakenly believe they can specify `https:` and somehow (magically) get a HTTPS server.  They might also mistakenly try to provide a path part.

On reflection, I wonder if:
```
prometheus.metrics.reporter.listener=[host]:port
```
would be better.

Reply:
jmx_exporter added support for https since a few releases already. Also in environments handling sensitive data it's often required, due to regulations, to use encryption on all data links. 

I did not include it in this proposal as today Strimzi does not support it with jmx_exporter. But since we expect the reporters to be usable outside Strimzi, I think at some point we will add support for https.